### ISSUE TITLE
Trackpoint/Waypoint-Timestamp Issue

### ISSUE THREAD

**Issue by rneppi** [zdila/gpx-animator#353]
There is a GPX-file with a track and with waypoints. The waypoints have timestamps. If one trackpoint and one waypoint have exact the same timestamp, in the movie jumps the track-line at this waypoint to the first waypoint (or trackpoint?) and then back, what is wrong.

Testet with two GPX-Files, where the waypoints added with several apps (Routeconverter and GPX-Editor). That means, GPX-file is not the problem.

System: macOS 10.15.7
GPX-Animator 1.6.1

**Comment by McPringle** [comment]
Thank you for using GPX Animator and also reporting bugs you notice. To recreate the bug, I need the GPX file and the GPX Animator configuration (File > Save As). You can compress the files as a ZIP file and attach them to this GitHub issue. If you don't want to upload the files to GitHub, you can also email them to support@gpx-animator.app. Please mention Issue #353 in the subject line.

**Comment by rneppi** [comment]
Thanks for looking at this thing.
I sent you an E-Mail with example-files.

**Comment by rneppi** [comment]
Sorry, didn't you got the Mail? Should I send it again?

**Comment by McPringle** [comment]
No, it was my fault, it got lost between a bunch of automated build server mails. I have your files and will investigate your problem.
